codelabs/grpc-java-opentelemetry/start_here/src/main/java/io/grpc/codelabs/opentelemetry/OpenTelemetryServer.java

```diff
@@ -86,7 +86,24 @@ public static void main(String[] args) throws IOException, InterruptedException
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    // CODELAB HINT : Add code to register GrpcOpenTelemetry here.
+    // CODELAB HINT : Add code to create Prometheus exporter here. The default
+    // port has been initialized for you in a variable i.e prometheusPort.
```

Comment:
s/i.e/named/ - "i.e." means "in other words"

Reply:
Done